src/bin/pgcopydb/schema.c

```diff
@@ -971,14 +971,19 @@ struct FilteringQueries listSourceTablesSQL[] = {
 		"               ("
 		"                  select attnum, atttypid::integer, "
 		"                         format('%I', attname) as attname, "
-		"                         i.indrelid is not null as attisprimary "
+		"                         i.indrelid is not null as attisprimary, "
+		"						  col.is_generated = 'ALWAYS' as attisgenerated "
 		"                    from pg_attribute a "
 		"                         left join pg_index i "
-		"                                on i.indrelid = a.attrelid "
+		"                               on i.indrelid = a.attrelid "
 		"                               and a.attnum = ANY(i.indkey) "
 		"                               and i.indisprimary "
+		"					 	  left join information_schema.columns col "
+        "                    			on col.column_name = a.attname "
```

Comment:
That's a good way to get at the information too, but I believe you need to also make sure you're targeting the column from the right table in information_schema: what if we have multiple tables using the same column name?